### ISSUE TITLE
in-app-browser > IOS > TMS-90809: Deprecated API Usage - New apps that use UIWebView are no longer accepted.

### ISSUE THREAD

**Issue by DeanVanNiekerk** [ionic-team/ionic-native#3404]
**I'm submitting a ...**
[x] bug report

**Current behavior:**
Apple Store Connect revokes submitted App with the following message:
`TMS-90809: Deprecated API Usage - New apps that use UIWebView are no longer accepted. Instead, use WKWebView for improved security and reliability.`

**Expected behavior:**
Apple Store Connect should accept the App

**Steps to reproduce:**
Added a reference to the `@ionic-native/app-rate` and `@ionic-native/in-app-browser` build the App and submit to the App Store Connect.

**Related code:**

```
AppRate.preferences = {
            ...AppRate.preferences,
            simpleMode: true,
            storeAppURL: {
                android: "market://details?id=xxxxxx",
                ios: "150xxxxxxxx",
            }
        };
```

**Other information:**
The App was being accepted by the App Store Connect before the 1st April 2020, after this date it started failing.

I see from this article that we need to set `WKWebViewOnly` to `true` for the cordova plugins, but how do we do this through Capacitor?

https://cordova.apache.org/howto/2020/03/18/wkwebviewonly.html

**Side Note:** I have upgraded and am using Capacitor v2.
A snip of some of my deps.
```
 "dependencies": {
        ......
        "@capacitor/android": "2.0.2",
        "@capacitor/core": "2.0.2",
        "@capacitor/ios": "2.0.2",
        "@ionic-native/app-rate": "5.24.0",
        "@ionic-native/core": "5.24.0",
        "@ionic-native/in-app-browser": "5.24.0",
        "cordova-plugin-apprate": "1.5.0",
        "cordova-plugin-inappbrowser": "3.2.0",
        ...
}

**Comment by jcesarmobile** [comment]
update to Capacitor 2.1.0, it automatically sets that flag

**Comment by DeanVanNiekerk** [comment]
Issue resolved after upgrading to Capacitor 2.1.0, thank you! 👍
